05/28/2020 11:04:32 PM: ERROR [logger_base.py:29] Prueba error
05/28/2020 11:04:32 PM: INFO [logger_base.py:30] Prueba Info
05/28/2020 11:04:32 PM: DEBUG [logger_base.py:31] Prueba debug
05/28/2020 11:04:32 PM: WARNING [logger_base.py:32] Prueba warnign
